_non-cosmos/polygon/assetlist.json

```diff
@@ -277,6 +277,46 @@
           "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/usdt.svg"
         }
       ]
+    },
+    {
+      "description": "The PAGE token is used for actions in the PageDAO NFT literary ecosystem and for DAO governance.",
+      "type_asset": "erc20",
+      "address": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+      "denom_units": [
+        {
+          "denom": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+          "exponent": 0,
+          "aliases": []
+        },
+        {
+          "denom": "PAGE",
+          "exponent": 8
+        }
+      ],
+      "base": "0x9ceE70895726B0ea14E6019C961dAf32222a7C2f",
+      "name": "Page",
+      "display": "Page",
+      "symbol": "PAGE",
+      "traces": [
+        {
+          "type": "additional-mintage",
+          "counterparty": {
+            "chain_name": "ethereum",
+            "base_denom": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e"
+          }
+        }
+      ],
+      "logo_URIs": {
+        "png": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.png",
+        "svg": "https://raw.githubusercontent.com/cosmos/chain-registry/master/_non-cosmos/ethereum/images/page.svg"
+      },
+      "coingecko_id": "page",
+      "images": [
+        {
```

Comment:
Within this object, please add the `image_sync` pointer to the ethereum $PAGE asset. E.g.,:
```
"image_sync": {
  "chain_name": "ethereum",
  "base_denom": "0x60e683C6514Edd5F758A55b6f393BeBBAfaA8d5e"
}
```